datalad/crawler/nodes/tests/test_misc.py

```diff
@@ -47,6 +48,23 @@ def test_get_disposition_filename():
     eq_(output[0]['filename'], 'T1.nii.gz')
 
 
+def test_sink():
+    data = {'x': 'y', 'g': 'h', 'a': 'b'}
+    keys = ['x', 'a']
+
+    # no arguments
+    genempty = Sink(None, None)
+    eq_(list(genempty(data)), [{'a': 'b', 'x': 'y', 'g': 'h'}])
+
+    # if key for sunk data is specified
+    gen = Sink(None, 'result')
```

Comment:
Could just use keyword argument without explicit None for the first positional argument